api/go.mod

```diff
@@ -16,3 +23,5 @@ require (
 	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
 	sigs.k8s.io/yaml v1.1.0
 )
+
+replace sigs.k8s.io/kustomize/api/internal/getter/helper/url => ./helper/url
```

Comment:
Why adding this `replace`? I don't think it's needed.

Reply:
removed